.changes/v1.0.0/4-features.md

```diff
@@ -0,0 +1,4 @@
+* **New Resource:** `vcfa_nsxt_manager` to manage VMware Cloud Foundation Automation Tenant Manager
+  NSX-T Managers [GH-4]
+* **New Data Source:** `vcfa_nsxt_manager` to read VMware Cloud Foundation Automation Tenant Manager
+  NSX-T Managers [GH-4]
```

Comment:
Do we need the "Tenant Manager" or leaving it generic? This is a doubt I also had for `vcd_org` (I didn't put it). If so, should it be `vcfa_tm_nsxt_manager`?

Reply:
Changed